jenkins_job_wrecker/cli.py

```diff
@@ -125,6 +125,11 @@ def parse_args(args):
         action='store_true', default=None,
         help='show more output on the console'
     )
+    parser.add_argument(
+        '-r', '--recursive',
+        action='store_true', default=None,
+        help='recursively look through jobs in folders'
+    )
```

Comment:
How about we just make this the default behavior and don't provide a config option?

Reply:
I'll do this as well. Now that I'm thinking about it, it makes more sense to follow folders no matter what.